packages/toolpad-app/src/toolpad/AppEditor/StructureExplorer/index.tsx

```diff
@@ -120,7 +120,7 @@ function RecursiveSubTree({
 
   if (children.length) {
```

Comment:
Let's not rely on implicit conversion. 

```suggestion
  if (children.length > 0) {
```